packages/web/src/common/store/user-list/utils.ts

```diff
@@ -72,6 +74,7 @@ export function createUserListProvider<T, U = void>({
     currentPage: number
     pageSize: number
   }) {
+    yield* call(waitForBackendSetup)
```

Comment:
I think it would be a little better to always have the `waitForBackendSetup` and `waitForAccount` lines together if they are both needed